tests/MenderAPI/deployments.py

```diff
@@ -173,6 +173,31 @@ def check_expected_status(
                 % (expected_status, deployment_id, max_wait)
             )
 
+    def check_not_in_status(
+        self, expected_status, deployment_id, max_wait=60 * 60, polling_frequency=0.2
+    ):
+        timeout = time.time() + max_wait
+
+        while time.time() <= timeout:
+            data = self.get_status(status=expected_status)
+
+            for deployment in data:
+                if deployment["id"] == deployment_id:
+                    time.sleep(polling_frequency)
+                    continue
+            else:
+                logger.info(
+                    "left deployment status (%s) as expected for: %s"
+                    % (expected_status, deployment_id)
+                )
+                return
+
+        if time.time() > timeout:
```

Comment:
Do you really need the if here?
I think you can only reach this point if `time.time() > timeout`.

Reply:
It is preexisting, actually, I copied this from the inverse function. But you're right, I believe I can remove it from both.

Reply:
I agree with this observation, the `if` condition could be removed from both functions.

Reply:
@tranchitella, @lluiscampos: Fixed!